src/node/rpc/member_frontend.h

```diff
@@ -1086,7 +1086,10 @@ namespace ccf
         // creations are in-flight and reading at the same version, all except
         // the first conflict and are re-executed. If we ever produce a
         // proposal ID which already exists, we must have a hash collision.
-        if (pm->has(proposal_id))
+        //
+        // This is current broken broken for BFT.
+        // https://github.com/microsoft/CCF/issues/2668
```

Comment:
What's the break?

Reply:
This is really #2136, and apart from the solutions proposed in the ticket, we could add a nonce to duplicate (or all proposals) in BFT as a short term hack.